src/Tizen.NUI/src/public/Visuals/VisualMaps.cs

```diff
@@ -82,6 +82,25 @@ public class VisualMap : IDisposable
         [EditorBrowsable(EditorBrowsableState.Never)]
         private Vector4 cornerRadius = null;
 
+        /// <summary>
+        /// The borderline width of the visual.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        private float? borderlineWidth = null;
+
+        /// <summary>
+        /// The borderline Color of the visual.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
No need to add `EditorBrowsable(EditorBrowsableState.Never)` attribute on private field

Reply:
Fix done.